src/PowerSimulations.jl

```diff
@@ -54,6 +54,7 @@ export construct_device!
 export construct_network!
 ## Op Model Exports
 export solve_op_model!
+export write_memory_results
```

Comment:
rename to `_write_variables` and don't export it.

Reply:
This is the function that writes the results in memory to a feather file (if no path was included as a kwarg or if the path was invalid). It's called within the development file I'm using, so it's not actually an internal function. Should I still rename it?

Reply:
Then let's call it `write_results` it is more intuitive.